cmd/main.go

```diff
@@ -65,7 +65,7 @@ import (
 
 var (
 	scheme               = runtime.NewScheme()
-	log                  = logger.Log
+	log                  logger.CustomLogger
```

Comment:
If we not use logger.Log for log in main, then this update of log will only apply to controller runtime log and main log. Other log in our pkg using logger.Log actually uses the log without zerolog? Is this by design?
```
log = logger.ZapCustomLogger(cf.DefaultConfig.Debug, config.LogLevel)
```